java-engine/src/main/java/io/getunleash/engine/UnleashEngine.java

```diff
@@ -27,25 +36,36 @@ public void takeState(String toggles) throws YggdrasilInvalidInputException {
         }
     }
 
-    public boolean isEnabled(String name, Context context) throws YggdrasilInvalidInputException {
+    public IsEnabledResponse isEnabled(String name, Context context) throws YggdrasilInvalidInputException {
```

Comment:
Let the SDK handle this, otherwise, we won't be able to inject a default value.

I'm still wondering if this method should accept an additional parameter to encapsulate the logic of handling isValid and reduce the exposure of internal classes, but I think I will only know when I start to integrate this into Java SDK.